Add `Featured Artist Track Updates: Winter 2025` news post (#12634)
ppy/osu-wiki: news/2025/2025-01-04-featured-artist-track-updates-winter-2025.md

---
layout: post
title: "Featured Artist Track Updates: Winter 2025"
date: 2024-01-04 20:00:00 +0000
---

If your New Year's resolution is to become a mapper, a solid first step would be experimenting with any of the **40** songs we added to our Featured Artist library today!

![](/wiki/shared/news/banners/featured-artist.jpg)

osu!'s [Featured Artist library](https://osu.ppy.sh/beatmaps/artists) has over **450** talented musicians at this point. While we usually reserve weekends for new artist announcements, we have a bunch of tracks from older artists to showcase, so... that's what we're doing.

> Any new songs are marked with a **NEW** icon on the Featured Artist listing

Reviewer: ```suggestion
Any new songs are marked with a **NEW** icon on the Featured Artist listing.
```

- **Midian - Malformation**
- **Midian - Midian's Lament**
- **Midian - Pixel Odyssey**
- **Midian - Polaris**
- **Midian - Space Exodium**
- **Midian - The Year of MIDIAN**
- **Midian - This Is OMMT4**

### Midian - Coagulate

<audio controls>
    <source src="https://assets.ppy.sh/artists/443/Songs/Midian_-_Coagulate.mp3" type="audio/mpeg">
</audio>

### Midian - Drifting Sideways

<audio controls>
    <source src="https://assets.ppy.sh/artists/443/Songs/Midian_-_Drifting_Sideways.mp3" type="audio/mpeg">
</audio>

## [nekodex](https://osu.ppy.sh/beatmaps/artists/1)

[![](https://assets.ppy.sh/artists/1/header.jpg)](https://osu.ppy.sh/beatmaps/artists/1)

[**nekodex**](https://osu.ppy.sh/beatmaps/artists/1) composed osu!'s theme song [circles!](). After being duct taped to a chair for a week straight, he remixed his own tune for the holiday season (which you almost definitely heard if you've played osu! in the last few weeks).

### nekodex - circle the halls

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2024-12-24-client-theme.mp4" type="video/mp4" preload="none">
    </video>
</div>

> 
> <audio controls>
>     <source src="https://assets.ppy.sh/artists/1/osu!%20themes/nekodex_-_circle_the_halls.mp3" type="audio/mpeg">
> </audio>

Reviewer: entire song is in the video

```suggestion
```

## [RINYA](https://osu.ppy.sh/beatmaps/artists/262)

[![](https://assets.ppy.sh/artists/262/header.jpg)](https://osu.ppy.sh/beatmaps/artists/262)

[**RINYA**](https://osu.ppy.sh/beatmaps/artists/262) recently rebranded from their previous alias **Mitsuki**, and in the process, they handed **4** high-paced tracks to osu! community:

### RINYA - BUCHIAGE Fireworks

<audio controls>
    <source src="https://assets.ppy.sh/artists/262/Songs/RINYA_-_BUCHIAGE_Fireworks.mp3" type="audio/mpeg">
</audio>

### RINYA - ChallengeR

<audio controls>
    <source src="https://assets.ppy.sh/artists/262/Songs/RINYA_-_ChallengeR.mp3" type="audio/mpeg">
</audio>

### RINYA - Kyokugen BON Odori

<audio controls>
    <source src="https://assets.ppy.sh/artists/262/Songs/RINYA_-_Kyokugen_BON_Odori.mp3" type="audio/mpeg">
</audio>

### RINYA - Over the Dimensions

<audio controls>
    <source src="https://assets.ppy.sh/artists/262/Songs/RINYA_-_Over_the_Dimensions.mp3" type="audio/mpeg">
</audio>

## [seatrus](https://osu.ppy.sh/beatmaps/artists/185)

[![](https://assets.ppy.sh/artists/185/header.jpg)](https://osu.ppy.sh/beatmaps/artists/185)

> Anyone who's heard [**seatrus**](https://osu.ppy.sh/beatmaps/artists/185)'s hyper-intense music would know how perfect it is for tournaments. Both of these tracks were licensed for their presence in the tourney scene: one from *The Perennial* and another which we won't spoil yet.

Reviewer: "super" and "hyper" can generally be used as suffixes without a hyphen

```suggestion
Anyone who's heard [**seatrus**](https://osu.ppy.sh/beatmaps/artists/185)'s hyperintense music would know how perfect it is for tournaments. Both of these tracks were licensed for their presence in the tourney scene: one from *The Perennial* and another which we won't spoil yet.
```

## [soowamisu](https://osu.ppy.sh/beatmaps/artists/376)

[![](https://assets.ppy.sh/artists/376/header.jpg)](https://osu.ppy.sh/beatmaps/artists/376)

[**soowamisu**](https://osu.ppy.sh/beatmaps/artists/376) is an expert at producing terminally online underground breakcore tracks. If that description means nothing to you, congratulations and listen up:

### soowamisu - FIRUSS EXoTICA

<audio controls>
    <source src="https://assets.ppy.sh/artists/376/Songs/soowamisu_-_FIRUSS_EXoTICA.mp3" type="audio/mpeg">
</audio>

### soowamisu - NEVERMORE Tenshi no

<audio controls>
    <source src="https://assets.ppy.sh/artists/376/Songs/soowamisu_-_NEVERMORE_Tenshi_no.mp3" type="audio/mpeg">
</audio>

### soowamisu - REPENTANCE

<audio controls>
    <source src="https://assets.ppy.sh/artists/376/Songs/soowamisu_-_REPENTANCE.mp3" type="audio/mpeg">
</audio>

### soowamisu - seraphiiim_tears777(nOuse)

<audio controls>
    <source src="https://assets.ppy.sh/artists/376/Songs/soowamisu_-_seraphiiim_tears777nOuse.mp3" type="audio/mpeg">
</audio>

## [Tenchio](https://osu.ppy.sh/beatmaps/artists/265)

[![](https://assets.ppy.sh/artists/265/header.jpg)](https://osu.ppy.sh/beatmaps/artists/265)

> [**Tenchio**](https://osu.ppy.sh/beatmaps/artists/265) is a one-of-a-kind musician behind a complex combination of math rock, noise, and pure emotion. Two tracks are now part of his listing, one of which has a two vocal versions: one vocaloid and another human!

Reviewer: ```suggestion
[**Tenchio**](https://osu.ppy.sh/beatmaps/artists/265) is a one-of-a-kind musician behind a complex combination of math rock, noise, and pure emotion. Two tracks are now part of his listing, one of which has two vocal versions: one VOCALOID and another human!
```

</audio>

## [UNDEAD CORPORATION](https://osu.ppy.sh/beatmaps/artists/45)

[![](https://assets.ppy.sh/artists/45/header.jpg)](https://osu.ppy.sh/beatmaps/artists/45)

You've been living under a rock if you haven't played an [**UNDEAD CORPORATION**](https://osu.ppy.sh/beatmaps/artists/45) map. Best known for [Ekoro](https://osu.ppy.sh/users/284905)'s masterpiece [Everything will freeze](https://osu.ppy.sh/beatmapsets/158023), the band still outputs Touhou and original tunes on the regular. Their 4 newest tunes are now officially part of osu!, each with both instrumental and vocal versions:

### UNDEAD CORPORATION - Bloodstained Nocturne

<audio controls>
    <source src="https://assets.ppy.sh/artists/45/Songs/UNDEAD_CORPORATION_-_Bloodstained_Nocturne.mp3" type="audio/mpeg">
</audio>

### UNDEAD CORPORATION - Fear Me

<audio controls>
    <source src="https://assets.ppy.sh/artists/45/Songs/UNDEAD_CORPORATION_-_Fear_Me.mp3" type="audio/mpeg">
</audio>

### UNDEAD CORPORATION - God of Leaf

<audio controls>
    <source src="https://assets.ppy.sh/artists/45/Songs/UNDEAD_CORPORATION_-_God_of_Leaf.mp3" type="audio/mpeg">
</audio>

> ### UNDEAD CORPORATION - Obituaries We(')rE None...

Reviewer: ```suggestion
<!-- lint ignore no-heading-punctuation -->

### UNDEAD CORPORATION - Obituaries We(')rE None...
```
